pandas/indexes/multi.py

```diff
@@ -2078,6 +2078,37 @@ def difference(self, other):
             return MultiIndex.from_tuples(difference, sortorder=0,
                                           names=result_names)
 
+    def symmetric_difference(self, other, result_name=None):
+        """
+        Compute sorted set symmetric difference of two MultiIndex objects
+
+        Returns
+        -------
+        diff : MultiIndex
+        """
+        self._assert_can_do_setop(other)
+        other, result_name_update = self._convert_can_do_setop(other)
+
+        if result_name is None:
+            result_name = result_name_update
+
+        if self.equals(other):
+            return MultiIndex(levels=[[]] * self.nlevels,
```

Comment:
make a constructor that creates an empty ``MultiIndex._create_as_empty()`` and just call here for both of these